exporter/qradar/README.md

```diff
@@ -0,0 +1,57 @@
+# QRadaar Exporter
+
+The QRadar Exporter is designed for forwarding logs to a QRadar endpoint using either Syslog or File-based methods. This exporter supports customization of data export types and various configuration options to tailor the connection and data handling to specific needs.
```

Comment:
How does the file-based method work? Is it applicable to qradar?

Reply:
Removed File-based method